From ac88acf8c9ae673dc9520afcd749fc30370b9cb1 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Fri, 12 Sep 2014 18:06:35 -0400
Subject: [powerpc] Fix a typo in comments of va to pa conversion

Message-id: <1410545655-205645-167-git-send-email-dzickus@redhat.com>
Patchwork-id: 94177
O-Subject: [RHEL7 PATCH 166/626] powerpc: Fix a typo in comments of va to pa conversion
Bugzilla: 1127366
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1127366

commit b83941798c35f9cffba36927011df2b53c3884d8
Author: Vaishnavi Bhat <vaishnavi@linux.vnet.ibm.com>
Date:   Sun Oct 27 11:47:19 2013 +0530

    powerpc: Fix a typo in comments of va to pa conversion

    This patch fixes typo in comments virtual to physical
    address conversion.

    Signed-off-by: Vaishnavi Bhat <vaishnavi@linux.vnet.ibm.com>
    Signed-off-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/include/asm/page.h b/arch/powerpc/include/asm/page.h
index b9f4262..753c662 100644
--- a/arch/powerpc/include/asm/page.h
+++ b/arch/powerpc/include/asm/page.h
@@ -78,7 +78,7 @@ extern unsigned int HPAGE_SHIFT;
  *
  * Also, KERNELBASE >= PAGE_OFFSET and PHYSICAL_START >= MEMORY_START
  *
- * There are two was to determine a physical address from a virtual one:
+ * There are two ways to determine a physical address from a virtual one:
  * va = pa + PAGE_OFFSET - MEMORY_START
  * va = pa + KERNELBASE - PHYSICAL_START
  *
-- 
1.7.1